submissions/OwoeyeGbenga/landing_page/index.html

```diff
@@ -0,0 +1,146 @@
+<!doctype html>
```

Comment:
Great job here. It is better to put your js logic in an external file and reference it here. Please fix.

Reply:
Done!!